src/test/munit/new-instance-test-suite.xml

```diff
@@ -226,7 +226,7 @@ payload.wheels must equalTo('wheels')
 payload.description must contain(vars.errorMessage)
 ]]]></munit-tools:that>
             </munit-tools:assert>
-            <munit-tools:assert-equals actual="#[payload.cause.cause.target.class]"
```

Comment:
What is the reason for the proposed change in property?
Any change in the structure that could potentially break backward compatibility?

Reply:
same as above (W-14380221)

Reply:
@martina-tutokiova just for clarity, could you add an assert on the class name of all the exceptions in this cause-chain? What would be the class of the first and second `cause` in this navigation?

Reply:
@elcoya assertions improved, thanks